collator/src/lib.rs

```diff
@@ -425,8 +418,7 @@ fn run_collator_node<S, E, P, Extrinsic>(
 							outgoing,
 						);
 
-						let exit = inner_exit_2.clone();
-						tokio::spawn(future::select(res.boxed(), exit));
+						tokio::spawn(res.boxed());
```

Comment:
@cecton did you check that the node still exits?

Reply:
Yes there is a test for that but in the other branch. I will make another PR to add it now

Reply:
Okay, ty!